README.md

```diff
@@ -244,29 +256,20 @@ under a category in libcxx, or running a single test in `std` and `tr1`.
 
 ## Examples
 
-```
-:: This command will run all of the testsuites with verbose output.
-
-C:\STL\out\build\x64>ctest -V
-
-:: This command will also run all of the testsuites.
-
-C:\STL\out\build\x64>python tests\utils\stl-lit\stl-lit.py ..\..\..\llvm-project\libcxx\test ..\..\..\tests\std ..\..\..\tests\tr1
-
-:: This command will run all of the std testsuite.
-
-C:\STL\out\build\x64>python tests\utils\stl-lit\stl-lit.py ..\..\..\tests\std
-
-:: If you want to run a subset of a testsuite you need to point it to the right place in the sources. The following
-:: will run the single test found under VSO_0000000_any_calling_conventions.
-
-C:\STL\out\build\x64>python tests\utils\stl-lit\stl-lit.py ..\..\..\tests\std\tests\VSO_0000000_any_calling_conventions
-
-:: You can invoke stl-lit with any arbitrary subdirectory of a testsuite. In libcxx this allows you to have finer
-:: control over what category of tests you would like to run. The following will run all the libcxx map tests.
-
-C:\STL\out\build\x64>python tests\utils\stl-lit\stl-lit.py ..\..\..\llvm-project\libcxx\test\std\containers\associative\map
-```
+These examples assume that your current directory is `C:\Dev\STL\out\build\x64`.
+
+* This command will run all of the testsuites with verbose output.
+  + `ctest -V`
+* This command will also run all of the testsuites.
+  + `python tests\utils\stl-lit\stl-lit.py ..\..\..\llvm-project\libcxx\test ..\..\..\tests\std ..\..\..\tests\tr1`
+* This command will run all of the std testsuite.
+  + `python tests\utils\stl-lit\stl-lit.py ..\..\..\tests\std`
+* If you want to run a subset of a testsuite you need to point it to the right place in the sources. The following

```

Comment:
For some reason, my brain wants a comma here.  Feel free to ignore, as I'm not entirely sure it's grammatically required or even appropriate.  
```suggestion
* If you want to run a subset of a testsuite, you need to point it to the right place in the sources. The following
```